README.md

```diff
@@ -81,6 +81,7 @@ assert_eq!(
         image_size: Size::new(8, 8),
         image_data_len: 192,
         channel_masks: None,
+        image_data_top_down: false
```

Comment:
The README needs to be regenerated.

Reply:
I did, but It's stuck on something I don't quite understand yet on the change to the header to make it non-exhaustive.  (I'm pretty new to Rust)

Reply:
OK, sorry for that. We are using struct literals for `Header` in some tests, which @jamwaffles and me hadn't considered. Just remove the `non_exhaustive` argument from `Header`.

Reply:
Will do! :D